app/controllers/performance_tracking_controller.rb

```diff
@@ -5,8 +5,11 @@ def index; end
 
   private
 
+  # Users that can manage others are only allowed to access this page on staging.
+  # This is configured by the allow_user_managers_service_access feature flag.
   def require_supervisor!
     return if current_user.supervisor? && FeatureFlags.basic_user_roles.enabled?
+    return if FeatureFlags.allow_user_managers_service_access.enabled?
```

Comment:
This logic is repeated in the view. We could add a helper method in the controller so that it can be used here and in the controller.

https://apidock.com/rails/ActionController/Helpers/ClassMethods/helper_method

Would be good to write a test for this as well that exercises all of the possible options that are expressed in the logic here.

User stubbing to make change the feature flags + users that have different roles to check it.

Reply:
We do have tests that cover these different scenarios from [our previous pr](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/381). We only added the new scenarios (when a user manager can access the page on staging) in this PR. But happy to chat through